packages/api-request-builder/src/default-services.js

```diff
@@ -191,6 +191,23 @@ export default {
       features.queryExpand,
     ],
   },
+  orderEdits: {
+    type: 'orders-edits',
+    endpoint: '/orders/edits',
+    features: [
+      features.create,
+      features.update,
+      features.del,
+      features.query,
+      features.queryOne,
+      features.queryExpand,
+    ],
+  },
+  orderEditsApply: {
```

Comment:
To use the `apply` method, we need to create a new endpoint because it calls the endpoint using the `id` before the apply method can be used([See here](https://docs.commercetools.com/http-api-projects-order-edits#apply-an-orderedit)).